extensions/smallrye-graphql/deployment/pom.xml

```diff
@@ -88,6 +88,11 @@
             <artifactId>quarkus-elytron-security-properties-file-deployment</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-graphql-client</artifactId>
```

Comment:
You have to depend on the deployment (`quarkus-smallrye-graphql-client-deployment`) module here